frontend/src/containers/expenses/dashboard.jsx

```diff
@@ -88,23 +116,26 @@ class Dashboard extends Component {
               />
             </Col>
           </Row>
-          <Row gutter={16} style={{ display: 'flex', flex: 1 }}>
+          <Row gutter={16}>
             <Col
+              id="expenses-column"
               xs={{ span: 24, order: 0 }}
               md={{ span: 12, order: 0 }}
-              style={{ height: '100%' }}
             >
               <ExpensesTable
                 title={moment(new Date(this.state.start_date)).format('MMMM')}
                 options={{
                   start_date: this.state.start_date,
                   end_date: this.state.end_date,
                 }}
+                tableHeight={
+                  this.state.dashboardHeight - this.state.topRowHeight
```

Comment:
Should handle when the ref doesnt have a height yet i.e. before component is mounted. Getting a warning for `maxHeight: NaN`

e.g.
```javascript
(this.state.dashboardHeight - this.state.topRowHeight) || 0
```

Reply:
Good catch, added default value in commit d6934e3